app/webpack.common.ts

```diff
@@ -59,7 +59,6 @@ const commonConfig: webpack.Configuration = {
   ],
   resolve: {
     extensions: ['.js', '.ts', '.tsx'],
-    modules: [path.resolve(__dirname, 'node_modules/')],
```

Comment:
Without this fix this PR would not build, since the newer version of `react-transition-group` depends on `dom-helpers@^5.0.1`, which has a different API than `dom-helpers@3.x`.

With the current dependency versions, v3 gets installed in `app/node_modules/dom-helpers`, while v5 gets installed in `app/node_modules/react-transition-group/node_modules/dom-helpers`. Because of this WebPack config, the v3 version was always getting used by WebPack.

Note that this change can potentially have an impact on our bundle size, since we used to (wrongly) deduplicate npm dependencies by always using the version installed in the root `node_modules` folder.

I'm going to measure the impact on bundle size

Reply:
Results comparing bundle sizes of development with this branch:


| | development | this branch | difference | % |
|-|-|-|-|-|
| Main process | 253924 | 262277 | 🔼 8.3KiB | 🔼 3.12%  |
| Renderer process | 1794308 |1812048  | 🔼 17.7KiB | 🔼 0.97% |

We can mitigate substantially this increase by deduplicating compatible dependencies from our `yarn.lock`, like the following:

https://github.com/desktop/desktop/blob/3cab784e0cbaed1fa342ed969777b720c414cb49/app/yarn.lock#L615-L623

I did a quick test deduplicating dependencies with [`yarn-deduplicate`](https://github.com/atlassian/yarn-deduplicate) and with that the main process bundle size on this PR was the same as in master and the renderer process bundle size on this PR was only 7.3KiB bigger than master.